scopes/scope/sign/sign.main.runtime.ts

```diff
@@ -3,7 +3,6 @@ import { CLIAspect, CLIMain, MainRuntime } from '@teambit/cli';
 import { LoggerAspect, LoggerMain, Logger } from '@teambit/logger';
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part